developer_tests/location/run_tests.csh

```diff
@@ -58,22 +73,13 @@ foreach i ( $LOCLIST )
      echo
    else
 
-      ls -l location_test
-      ( ./location_test < test.in > $LOGDIR/runlog.$i.out ) || set FAILURE = 1
+     ( ./location_test < test.in > $LOGDIR/runlog.$i.out ) || set FAILURE = 1
 
-     echo
-     echo
      if ( $FAILURE ) then
-       echo "------------------------------------------------------------------"
```

Comment:
There are some formatting changes here ------, and print messages changed.  
May want to revert these to keep `test_dart.csh` formatting the same.

Reply:
I vote to make the output less cluttered. I have looked at lots of these and the formatting has actually been a bit confusing.
I have been trying to make the banner of `====` appear at the start and end of each 'run_dart.csh' and use banners of `----` to denote the start and end of tests within a directory ... (or something like that) ...

Reply:
leaving Nancy's less cluttered version as is.